packages/frontend/src/shared/ui/preloader/Preloader.module.scss

```
@@ -0,0 +1,39 @@
+.wrapper {
+  position: relative;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 80px;
+  height: 80px;
+}
+
+.preloader {
+  position: absolute;
+  width: 60px;
+  height: 60px;
+  background-image: url('/images/preloader/backgraund-r-80.svg');
```

Опечатка в названии картинки. Саму картинку тоже нужно переименовать
```suggestion
  background-image: url('/images/preloader/background-r-80.svg');
```